tree/ntuple/v7/src/RColumnElement.hxx

```diff
@@ -240,102 +222,80 @@ static void CastZigzagSplitUnpack(void *destination, const void *source, std::si
       dst[i] = static_cast<SourceT>((val >> 1) ^ -(static_cast<SourceT>(val) & 1));
    }
 }
+} // namespace
 
-} // anonymous namespace
+// anonymous namespace because these definitions are not meant to be exported.
+namespace {
 
-namespace ROOT {
-namespace Experimental {
-namespace Internal {
+using ROOT::Experimental::EColumnType;
```

Comment:
You may consider also `using ROOT::Experimental::Internal::RColumnElementBase` to avoid some churn